sdk/storage/azure-storage-blob-stress/scripts/stress-run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
Are these two scripts used anywhere? Or are they just for local testing?

Reply:
In that case it won't work, since the output path for the dev cert is `/mnt/outputs` which is specific to the stress container environment.